### DISCUSSION TITLE
oidc authservice back off 문제

### DISCUSSION THREAD

**Discussion by ChaejinE** [mlops-for-all/mlops-for-all.github.io#113] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/69780812/211990633-c6514fda-2d94-425d-9628-a79960b3494b.png)

- 환경 : mac m1 silicon
  - k3s 1.25.4
  - kubeflow : 1.6

```shell
kustomize build common/oidc-authservice/base | kubectl apply -f -
```
- 에서 해당 part에서 계속 crashloopbackoff가 걸리는데 짐작되시는 부분있을까요 ?

**Comment by zamonia500** [comment, marked as answer]
authservice-0의 log를 첨부해주시면 확인에 도움이 될 것 같습니다!

**Reply by ChaejinE** [reply]
아네 감사합니다.. 알아보니 amd에서 만든 pod여서 그런지 arm64에서는 돌아가지 않는 것으로 보입니다 .. log를 보니 oidc 컨테이너 안에서 실행해야되는 파일이 실행할 수 없다는 로그만 나왔었습니다.. (오래되서 기록을 못했네요 ㅠ)

**Reply by zamonia500** [reply]
그렇다면 직접 arm arch로 다시 image를 빌드하시면 됩니다. 생각보다 간단합니다!
https://github.com/arrikto/oidc-authservice#build

Build, 원하는 repository에 push, 하도 kubeconfig을 수정하면 됩니다!

**Reply by ChaejinE** [reply]
buildx 로 arm 에서 돌아가도록 빌드 후 커스텀 이미지로 바꿨는데 실패하긴했습니다 ㅠ .. buildx랑 다를려나.. 무튼 감사합니다. 나중에 해봐야곘어요
